companion/gateway/services/memory_manager.py

```
@@ -0,0 +1,946 @@
+                    None,
+                    lambda: self.qdrant.get_collections()
+                )
+            except Exception as e:
+                self.logger.exception("Qdrant health check failed")
+                return False
+
+            # Check embedding client availability
+            try:
+                # Simple test embedding to verify client is working
+                await self.embedding_client.embed_text("health check test")
+            except Exception as e:
```

Ruff (0.13.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)